test/beef_test.go

```diff
@@ -34,17 +34,27 @@ func TestBeef(t *testing.T) {
 			address, privateKey := getNewWalletAddress(t)
 			dstAddress, _ := getNewWalletAddress(t)
 
-			txID := sendToAddress(t, address, 0.001)
+			txID := sendToAddress(t, address, 0.002)
 			hash := generate(t, 1)
 
-			beef, tx := prepareBeef(t, txID, hash, address, dstAddress, privateKey)
+			beef, middleTx, tx := prepareBeef(t, txID, hash, address, dstAddress, privateKey)
 
 			body := api.POSTTransactionJSONRequestBody{
 				RawTx: beef,
 			}
 
+			callbackReceivedChan := make(chan *api.TransactionStatus)
+			callbackErrChan := make(chan error)
+
+			callbackUrl, token, shutdown := startCallbackSrv(t, callbackReceivedChan, callbackErrChan, nil)
```

Comment:
Please move `startCallbackSrv(...)` to `utilis.go`

Reply:
Done